<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GammaRay::PropertyExtendedEditor</class>
 <widget class="QWidget" name="GammaRay::PropertyExtendedEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>392</width>
    <height>23</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="valueLabel"/>
   </item>
   <item>
    <widget class="QPushButton" name="editButton">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
